src/Layouts/MainWindow.vala

```diff
@@ -23,11 +23,46 @@
 public class Photostat.Layouts.MainWindow : Gtk.Grid {
     public weak Photostat.Window window { get; construct; }
 
+    public Layouts.MainCanvas main_canvas;
+    public Layouts.RightSideBar right_sidebar;
+    public Layouts.PropertiesBar properties_bar;
+    public Layouts.ToolsBar tools_bar;
+
+    public Gtk.Paned main_pane;
+    public Gtk.Paned child_pane;
+
+    public Gtk.Grid left_sidebar;
+
+
     public MainWindow (Photostat.Window window) {
         Object (window: window);
     }
 
     construct {
-        // Add sidebars and main canvas here
+        right_sidebar = new Layouts.RightSideBar (window);
+        properties_bar = new Layouts.PropertiesBar (window);
+        main_canvas = new Layouts.MainCanvas (window);
+        tools_bar = new Photostat.Layouts.ToolsBar (window);
+
+        main_pane = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
+        child_pane = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
+
+        left_sidebar = new Gtk.Grid ();
+        left_sidebar.get_style_context ().add_class ("sidebar");
```

Comment:
I think we should use `Gtk.STYLE_CLASS_SIDEBAR` instead of our own CSS

Reply:
The background color

Reply:
Are you talking about the background color of the sidebar?? 

Nvm, I'm merging this. If we want to use the `Gtk.STYLE_CLASS_SIDEBAR` class, it should be a small fix, and should break the code.